models/host.tsp

```diff
@@ -1,12 +1,14 @@
 import "@typespec/json-schema";
 import "./common.tsp";
+import "./dns.tsp";
 
 using TypeSpec.JsonSchema;
 
 namespace rpp.host {
 
     model HostInfoObj {
         name: string;
+        ...rpp.dns.DNS
```

Comment:
This one shall be added to HostInfoAttr not HostInfoObj.

/hosts is a collection of Host model (below) which extends HostInfoAttr.

Reply:
Added in the latest commits
* model `HostAttr` does the extension of `HostInfoAttr`
* model `DNSHostObj` is the dns alternative to the Name+Glue host object 
* a bit more documentation